<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>210</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Query databases</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="btSelectAll">
       <property name="toolTip">
        <string>Select all items</string>
       </property>
       <property name="text">
        <string>Select &amp;all</string>
       </property>
       <property name="icon">
        <iconset resource="ui.qrc">
         <normaloff>:/icons/images/select-all.svg</normaloff>:/icons/images/select-all.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btSelectNone">
       <property name="toolTip">
        <string>Select nothing</string>
       </property>
       <property name="text">
        <string>Select &amp;none</string>
       </property>
       <property name="icon">
        <iconset resource="ui.qrc">
         <normaloff>:/icons/images/select-none.svg</normaloff>:/icons/images/select-none.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btSelectInvert">
       <property name="toolTip">
        <string>Invert selection</string>
       </property>
       <property name="text">
        <string>&amp;Invert selection</string>
       </property>
       <property name="icon">
        <iconset resource="ui.qrc">
         <normaloff>:/icons/images/select-invert.svg</normaloff>:/icons/images/select-invert.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::VLine</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btSaveSelection">
       <property name="text">
        <string>Save selection</string>
       </property>
       <property name="icon">
        <iconset resource="ui.qrc">
         <normaloff>:/icons/images/document-save.svg</normaloff>:/icons/images/document-save.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Databases</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QListWidget" name="lstDatabases">
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="tabKeyNavigation">
         <bool>true</bool>
        </property>
        <property name="showDropIndicator" stdset="0">
         <bool>false</bool>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::MultiSelection</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="ui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>340</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>340</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
